workflows/ncn/worker/worker.wipe-and-reboot.yaml

```
@@ -140,7 +140,7 @@ tasks:
             
             # wait for boot
             counter=0
-            source /etc/cray/upgrade/csm/myenv
+            source /etc/cray/upgrade/csm/myenv || true
```

Will this mask nefarious things in `myenv`? Would it be better to do the following?

```bash
[ -f /etc/cray/upgrade/csm/myenv ] && source /etc/cray/upgrade/csm/myenv
```

---

I think your modified version will fail if the code runs with set -e (which I assume it does, because of the `|| true`.

I think your change makes sense, but it would need to be modified to something like:

```suggestion
[ ! -f /etc/cray/upgrade/csm/myenv ] || source /etc/cray/upgrade/csm/myenv
```